.changes/v2.15.0/443-features.md

```diff
@@ -0,0 +1,7 @@
+* Added new structure `AnyEdgeGateway` which supports retreving both types of Edge Gateways (NSX-V
+  and NSX-T) with methods `AdminOrg.GetAnyEdgeGatewayById`, `Org.GetAnyEdgeGatewayById`,
+  `AnyEdgeGateway.IsNsxt`, `AnyEdgeGateway.IsNsxv`, `AnyEdgeGateway.GetNsxtEdgeGateway` [GH-443]
```

Comment:
_(This comment is just sharing thoughts with no need for action)._

To be honest, the chosen `Any` naming had to sink in with me :) Mainly, because "any" implies a "random" item. 

The true meaning here is a "general edge gateway". However, "general" is a long word, so I'm not suggesting that. In fact, `Any` is laconic and nice, it's just a bit confusing when you see it for the first time.

Reply:
I also was triggered by `Any`.  Possible variations common, general. Or make renaming to EdgeGateway which can by NsxvEdgeGateway or NsxtEdgeGateway

Reply:
> Or make renaming to Or make renaming to EdgeGateway which can by NsxvEdgeGateway or NsxtEdgeGateway which can by NsxvEdgeGateway or NsxtEdgeGateway

How expensive would switching to pure `EdgeGateway` (without `Any` prefix) be? Is it in use already?

> Possible variations common, general.

Good point, though I'd vote "general" over "common" if we end up going this route.

Reply:
> I also was triggered by `Any`. Possible variations common, general. Or make renaming to EdgeGateway which can by NsxvEdgeGateway or NsxtEdgeGateway

I'd vote for this, though not sure how complicated it would be (in terms of refactoring code here and there).

Another proposal here, naming these ones `AdminOrg.GetNsxtOrNsxvEdgeGatewayById`, `Org.GetNsxtOrNsxvEdgeGatewayById` and the struct `NsxtOrNsxvEdgeGateway`.

Reply:
Renamed structure and methods to contain `AnyType`instead of only `Any`

Reply:
Thank you. There are some (I see 18) places left to rename: you can find them by doing "anye" search across all the PR.

Reply:
True. I was overly impressed with refactoring but missed manual comments. Did a search and replace. grep doesn't find more occurrences.